k8s/aat/common/divorce/div-cos.yaml

```diff
@@ -34,6 +34,7 @@ spec:
         FEATURE_RESP_SOLICITOR_DETAILS: "true"
         SPRING_MAIL_PROPERTIES_MAIL_SMTP_STARTTLS_ENABLE: false
         SCHEDULER_SEND_UPDATED_CASES_TO_ROBOTICS_ENABLED: true
+        BULKACTION_MINCASES: 5
```

Comment:
Is there a COS PR to use this?

Reply:
Because I can't find `BULKACTION_MINCASES` in COS

Reply:
https://github.com/hmcts/div-case-orchestration-service/blob/7afa572ed652bc1f82c7018da3485178e30d8c2d/src/main/java/uk/gov/hmcts/reform/divorce/orchestration/tasks/BulkCaseCreate.java#L51 - as it minimum 30- difficult on AAT for testing.

Reply:
> Because I can't find `BULKACTION_MINCASES` in COS

https://github.com/hmcts/div-case-orchestration-service/blob/31e4e69104ad19ad78a449465fe95f94e9da95a7/src/main/resources/application.yml#L358